cmip6_downscaling/analysis/analysis.py

```diff
@@ -0,0 +1,183 @@
+import os
+
+import cartopy.crs as ccrs
+import fsspec
+import matplotlib.pyplot as plt
+import pandas as pd
+import papermill as pm
+import xarray as xr
+from azure.storage.blob import BlobServiceClient, ContentSettings
+from prefect import task
+
+from cmip6_downscaling.data.cmip import convert_to_360
+
+from .qaqc import make_qaqc_ds
+
+connection_string = os.environ.get("AZURE_STORAGE_CONNECTION_STRING")
+fs = fsspec.filesystem('az', connection_string=connection_string)
```

Comment:
Do we need to do this at the module level? I think with our recent changes to the config/runtimes, this shouldn't be necessary.

Reply:
we now load in via `os.getenv`